src/snowflake/snowpark/session.py

```diff
@@ -958,6 +980,13 @@ def _resolve_packages(
                             "by ORGADMIN to use Anaconda 3rd party packages. Please follow the instructions at "
                             "https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages.html#using-third-party-packages-from-anaconda."
                         )
+                    if not upload_custom_packages:
+                        raise RuntimeError(
```

Comment:
Should we highlight this error message change in the release note?

Reply:
Sure, which changelog section do I add it under?